### ISSUE TITLE
Appimage icon with custom PNG

### ISSUE THREAD

**Issue by Kyanoxia** [sarweshparajuli/nativefier-appimage#17]
More of a feature request:

### Name Editing
It would be great if we had support for editing source before you package it into an AppImage.  Mainly to change the name and icon of the Application, for example if you try to make a package out of `monarch-app.aop.com`, you will see when you launch it, the application name isn't your `--name` flag, it is named `monarch-nativefier-[numbers and letters]`
![image](https://user-images.githubusercontent.com/61282566/170704727-75e587c9-e222-487e-9bf1-a7ff51b2a016.png)

### Icon Editing
With the Nativefier package, you can edit the source of the application to change your icon (and name too).  Again, with the Monarch example, the Icon when you launch it looks very blurry and sad.  I have made an alternative icon that I would like to use, but you cannot do that with this package.
![image](https://user-images.githubusercontent.com/61282566/170706752-0f2d9829-33e1-4943-9ee0-f3d8ee3bdde4.png)

So basically, my request is a delay between making the source, and compiling it to the AppImage.

**Comment by Kyanoxia** [comment]
Okay so after further research I found out that you can extract appimages and then recompile them.  I've used that technique to change the files around and can now successfully change the icon, name and other things.
